app/views/tag/_tagging.html.erb

```diff
@@ -3,7 +3,7 @@
 <p>
   <span id="tags">
     <% @node.tags.each do |tag| %>
-      <span id="tag_<%= tag.id %>" class="label label-primary">
+      <span id="tag_<%= tag.id %>" class="label label-primary" data-toggle="tooltip" data-html="true" title="created by <strong><%= tag.author.username %></strong> <%= time_ago_in_words(Time.at(tag.drupal_node_community_tag.first.date)) %> ago">
```

Comment:
The first part of this line would stay the same. But `tag.drupal_node_community_tag.first.date` could then become `tag.date` since we'd already be inside the `community_tag`. Sorry this seems a bit convoluted... it's:

* `nodes` have many `community_tags` which have an `author` and `date`
* `community_tags` belong to `tags` which have a `name`

This way, we can easily collect unique uses of each tag name. But it made our lives more difficult here. Again, sorry for my oversight here, I had meant to help!